### PR TITLE
Update test to run network check in both rootless and rootfull mode

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@Luap99 PTAL

**Comment by Luap99** [comment]
Will this fail on my laptop when there is no containers user setup in `/etc/sub?id` ?

**Comment by rhatdan** [comment]
Good question. Probably.

**Comment by rhatdan** [comment]
@Luap99 PTAL

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

File: test/system/500-networking.bats [hunk 1]
```diff
@@ -106,7 +105,9 @@ load helpers
         # remote IP is not 127.0.0.1 (podman PR #9052).
         # We could get more parseable output by using $NCAT_REMOTE_ADDR,
         # but busybox nc doesn't support that.
-        run_podman run -d --userns=keep-id $network_arg -p 127.0.0.1:$myport:$myport \
+        userns="--userns=keep-id"
+        is_rootless || userns="--uidmap=0:1111111:65536 --gidmap=0:1111111:65536"
```
Comment (edsantiago) [inline test/system/500-networking.bats:109]: DAE find the `||` form hard to read in this context? It takes my brain an extra second or two to interpret it. I know it takes up lines, but would anyone object to an `if/else`?

**Review by edsantiago — APPROVED** [review]
One nit, not worth re-pushing for; just a question to ponder for next time.

/lgtm

File: test/system/500-networking.bats (hunk 1 above)
Comment (rhatdan) [reply test/system/500-networking.bats:109]: I am fine with that, but the `is_rootless ||` is used in multiple places, so I went with that.

**Pull request merged**